spec/factories/courses.rb

```diff
@@ -4,5 +4,6 @@
     description { "Brake systems description" }
     code { "AA11" }
     units { "3.5" }
+    organization
```

Comment:
Same here -- is organization required? From the ticket it sounds like it can be optional, so let's leave it out to avoid creating unnecessary db records during the test suite.

Reply:
I was just going off of the comment in the ticket: "**Most** of our courses will have a school of some sort attached to them" (bold mine). @Pcushing do you have input?